src/shared/api/errors.ts

```
@@ -0,0 +1,16 @@
+export class ApiError extends Error {
+  code: string;
+  status: number;
+
+  constructor(message: string, code: string = '', status: number = 0) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)